### DISCUSSION TITLE
Welcome to NameSys.eth

### DISCUSSION THREAD

**Discussion by 0xc0de4c0ffee** [namesys-eth/ccip2-eth-resources#8] [General]
🙏 @eth-limo & @tomlightning WC. 
Take a look around discussion and ccip2.eth related repos. 
Feel free to use this discussion for any related topics/questions.

We also have half stack spread over https://github.com/dostr-eth side. 🖖

**Comment by sshmatrix** [comment]
Welcome @tomlightning and @eth-limo 🥳
